src/main/resources/api/service_openapi.yaml

```diff
@@ -702,6 +713,10 @@ components:
       items:
         $ref: '#/components/schemas/RoleBinding'
 
+    MemberEmail:
```

Comment:
It might make sense (later) to have a `ValidEmailAddress` type instead of string, to indicate that you've got a real email address (for tighter typing). Might be on the cute side though.